### PR TITLE
VisualEditor config for several wikis T3683

### PR DESCRIPTION
**Pull request by Amanda-Catherine**

### CODE REVIEW

File: LocalSettings.php [hunk 1]
```diff
@@ -4411,6 +4411,99 @@
 			NS_MAIN => true,
 			NS_USER => true,
 		),
+		'wmgVisualEditorAvailableNamespaces' => array(
```
Comment (MacFan4000) [inline LocalSettings.php]: This array isn’t closed

Comment (Amanda-Catherine) [reply LocalSettings.php]: @MacFan4000 looks fine to me.. I’m confused

Comment (JohnFLewis) [reply LocalSettings.php]: It’s an array of the setting within an array of the setting.

File: LocalSettings.php [hunk 2]
```diff
@@ -4572,7 +4673,7 @@
 			'random_poll' => false,
 		),
 	),
-);
```
Comment (JohnFLewis) [inline LocalSettings.php]: Shouldn’t have been touched

File: LocalSettings.php (hunk 1 above)
Comment (Amanda-Catherine) [reply LocalSettings.php]: @JohnFLewis I am still very confused. I’ve given you temporary access to my repo fork - please fix it for me. Sorry, but I just don’t get what I screwed up.

**Pull request merged**